WORKSPACE

```diff
@@ -83,10 +101,13 @@ local_repository(
 ####################################
 # Load and install our dependencies downloaded above.
 
-load("@build_bazel_rules_nodejs//:defs.bzl", "check_bazel_version", "node_repositories", "yarn_install")
+load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories", "yarn_install", "check_rules_nodejs_version")
 
 node_repositories(package_json = ["//:package.json"])
 
+# 0.11.3: proper module resolution & check_rules_nodejs_version
+check_rules_nodejs_version("0.11.3")
```

Comment:
Can't `check_rules_typescript_version` do this for us?

Reply:
It can and it does as well in ts_setup_workspace. Would you rather not have it checked here?

Reply:
yeah users shouldn't need to do transitive checks